src/css/custom.css

```diff
@@ -159,3 +159,92 @@ html[data-theme='dark'] .main-wrapper aside {
 .table-of-contents {
   font-size: 14px;
 }
+
```

Comment:
I think much of this should move to SdkPage.module.css in the same directory as the component - that way it's located close to the component that it references, and doesn't have to be global in scope. For an example of how this works, check out the components here: https://github.com/opcr-io/opcr-website/tree/main/src/components

FeatureDetails.js defines some components, and FeatureDetails.module.css defines the CSS. You can import the styles, like on line 3:

import styles from './FeatureDetails.module.css';

and then refer to them, like on line 66:

<Svg className={styles.featureDetailsSvg} alt={title} />